languages/java/exercises/concept/lists/.docs/after.md

```diff
@@ -0,0 +1,55 @@
+The Java lists are a collection of items, ordered and that don't have a fixed size. They can grow dynamically.
+
+It is an interface, so we can't instantiate it directly. There are several implementation in the base library, for example :
+
+```java
+List<T> arraylist = new ArrayList<T>();
+List<T> linkedlist = new LinkedList<T>();
+List<T> vector = new Vector<T>();
+List<T> stack = new Stack<T>();
+```
```

Comment:
Do you intend to have separate exercises for the other list types (like `Vector` and `Stack`)? If not, it might be useful to briefly discuss when to use which list type.